From d7614cd72dc1680801936c1d468704426c2d142a Mon Sep 17 00:00:00 2001
From: Hugo Villeneuve <hvilleneuve@dimonoff.com>
Date: Thu, 16 Jul 2026 17:12:10 -0400
Subject: [PATCH 05405/15333] serial: core: display 32 for MEM32 and MEM32BE
 UPIO types

Displaying "32be" is wrong for UPIO_MEM32. Fix and simplify by displaying
"32" for MEM32 and MEM32BE UPIO types.

Fixes: 86305190f307 ("serial: uniformize serial port I/O infos display")
Signed-off-by: Hugo Villeneuve <hvilleneuve@dimonoff.com>
Link: https://patch.msgid.link/20260716211216.2583291-1-hugo@hugovil.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/tty/serial/serial_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/serial_core.c b/drivers/tty/serial/serial_core.c
index 06d7f44643d6..edd1e7be2a5c 100644
--- a/drivers/tty/serial/serial_core.c
+++ b/drivers/tty/serial/serial_core.c
@@ -2492,7 +2492,7 @@ static const char *uart_get_mmio_width(struct uart_port *port)
 		return "16";
 	case UPIO_MEM32:
 	case UPIO_MEM32BE:
-		return "32be";
+		return "32";
 	case UPIO_AU:
 	case UPIO_MEM:
 	default:
-- 
2.55.0

